javers-spring-boot-starter-sql/src/main/java/org/javers/spring/boot/sql/JaversSqlAutoConfiguration.java

```diff
@@ -131,4 +132,10 @@ public JaversAuditableAspect javersAuditableAspect(Javers javers, AuthorProvider
     public JaversSpringDataAuditableRepositoryAspect javersSpringDataAuditableAspect(Javers javers, AuthorProvider authorProvider) {
         return new JaversSpringDataAuditableRepositoryAspect(javers, authorProvider, commitPropertiesProvider());
     }
+
+    @Bean
+    @ConditionalOnProperty(name = "javers.springDataAuditableRepositoryAspectEnabled", havingValue = "true", matchIfMissing = true)
+    public JaversSpringDataJpaAuditableRepositoryAspect javersSpringDataJpaAuditableRepositoryAspect(Javers javers, AuthorProvider authorProvider) {
```

Comment:
jeśli nowy aspekt będzie extendowal JaversSpringDataAuditableRepositoryAspect to chyba nie potrzebujemy drugiego beana?

Reply:
Caused by: org.springframework.aop.framework.AopConfigException: [org.javers.spring.auditable.aspect.springdata.JaversSpringDataJpaAuditableRepositoryAspect] cannot extend concrete aspect [org.javers.spring.auditable.aspect.springdata.JaversSpringDataAuditableRepositoryAspect]